viam-cartographer/src/carto_facade/carto_facade.h

```diff
@@ -82,7 +82,7 @@ typedef struct viam_carto_sensor_reading {
     bstring sensor;
     // TODO: change to void* and a size
     bstring sensor_reading;
-    unsigned long long sensor_reading_time_unix_micro;
+    int64_t sensor_reading_time_unix_micro;
```

Comment:
@kim-mishra I decided to change this type to be int64_t given that that 

`time.Now().UnixMicro()` https://pkg.go.dev/time#Duration.Microseconds returns an int64 signed 64 bit integer, and previously the C type was an unsigned 64 bit integer. I also figured that using `int64_t` would be more semantically meaningful than `long long` for those not familiar with C.